site/content/en/docs/Getting Started/create-webhook-fleetautoscaler.md

```diff
@@ -307,14 +307,30 @@ Every webhook that you wish to install a trusted certificate will need to go thr
 openssl genrsa -out webhook.key 2048
 ```
 
-Once the key is created, you’ll generate the certificate signing request, use valid hostname which is `autoscaler-tls-service.default.svc` as `Common Name (eg, fully qualified host name)` when prompted:
+Next create configuration file `cert.conf` for the certificate signing request:
 ```
-openssl req -new -key webhook.key -out webhook.csr
+[req]
+distinguished_name = req_distinguished_name
+req_extensions = v3_req
+prompt = no
+[req_distinguished_name]
+CN = autoscaler-tls-service.default.svc
+[v3_req]
+keyUsage = digitalSignature
+extendedKeyUsage = serverAuth
+subjectAltName = @alt_names
```

Comment:
You can also use the existing configurations in openssl.cnf and only set SAN. Here is an example of doing it in the old version of Agones documentations: https://1-4-0.agones.dev/site/docs/advanced/allocator-service/#server-tls-certificate

Reply:
oooh!

I have no strong opinions either way. Whichever is easier to explain?

Do we feel like having a file better, because it's easier for automation tools to utilise?

Reply:
I am happy to change it to use the `/etc/ssl/openssl.cnf`, but my thinking is that if you use that we are appending extra config to a fairly chunk example file. Whereas if we use a custom `cert.conf` it is easier for people to follow what is actually happening.

Reply:
> I am happy to change it to use the /etc/ssl/openssl.cnf, but my thinking is that if you use that we are appending extra config to a fairly chunk example file. Whereas if we use a custom cert.conf it is easier for people to follow what is actually happening.

Yeah, I would agree. Also wouldn't want people editing/thinking they have to edit the `/etc/ssl/openssl.cnf` file either.

Reply:
Shall I keep it as is then?